pkgs/tools/video/harvid/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, ffmpeg, libjpeg, libpng, pkg-config }:
+stdenv.mkDerivation {
+  pname = "harvid";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "x42";
+    repo = "harvid";
+    rev = "d71921ba52efd1362973df850213eebe97a7ecc6";
+    sha256 = "0l1plfsfh2ixhlzg3hqqvjj42z7g422718a9kgbh7b4p882n71x7";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ ffmpeg libjpeg libpng ];
+
+  installPhase = ''
+    runHook preInstall
+    make DESTDIR="$out" libdir="/lib" install
+    mkdir -p $out/bin
+    mv $out/usr/local/bin/* $out/bin
+    mv $out/usr/local/share $out/
+    rm -r $out/usr
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description =
+      "Decodes still images from movie files and serves them via HTTP";
```

Comment:
```suggestion
    description = "Decodes still images from movie files and serves them via HTTP";
```

Reply:
Omitting this formatting change as the current result is produced by `nixfmt`.

Reply:
nixfmt is not the official formatter for nixpkgs and we do not have a ~80 line length limit. It is totally fine to not linebreak a 120 or even 240 character URL and in this example the line break only stretches the file instead of making it more reable.